tex/alg-NT/classgrp.tex

```diff
@@ -701,15 +701,16 @@ \section{Computation of class numbers}
 First, we use a lemma that will help us with
 narrowing down the work in our cherry tree.
 \begin{lemma}[Ideals divide their norms]
+	\label{lemma:ideal_divides_norm}
 	Let $\kb$ be an integral ideal with $\Norm(\kb) = n$.
 	Then $\kb$ divides the ideal $(n)$.
 \end{lemma}
 \begin{proof}
 	By definition, $n = \left\lvert \OO_K / \kb \right\rvert$.
 	Treating $\OO_K/\kb$ as an (additive) abelian group and using Lagrange's theorem, we find
 	\[ 0 \equiv
-		\underbrace{\alpha + \dots + \alpha}_{n\text{ times}} = n\alpha
-		\pmod \kb \qquad\text{for all } \alpha \in \OO_K. \]
+		\underbrace{1 + \dots + 1}_{n\text{ times}} = n
+		\pmod \kb \qquad\text{for } 1 \in \OO_K. \]
```

Comment:
Actually this change may backfire... after all `(n)` is defined to be `{n \alpha | \alpha \in \OO_K}`. Guess I'll just revert it.